### PR TITLE
PyTorch RF (scikit-learn) for BCI drone control — metrics + model

### LINKED ISSUE #306: PyTorch: Random Forest - train a brainwave model
Using the provided data, obtain the success rates for Drone control model with the brain. Use PyTorch Random Forest Classifier.

Check on Wiki - adjust to PyTorch accordingly:
https://github.com/3C-SCSU/Avatar/wiki
Image

Do a pull request, adding the new PyTorch code implementation and trained model in the directory:

Image

### PR DESCRIPTION
**Pull request by Esma-Ali**
Adds a RandomForest baseline under:
`deep-learning-prediction/pytorch-rf/`

How to run (local; data kept private):
1) pip install -r deep-learning-prediction/pytorch-rf/requirements.txt
2) python deep-learning-prediction/pytorch-rf/rf_features_from_tree.py --root <raw_root> --out features.csv
3) python deep-learning-prediction/pytorch-rf/train_rf.py --train_csv features.csv --label_col label --out_dir deep-learning-prediction/pytorch-rf/artifacts --cv 5

Artifacts committed:
- deep-learning-prediction/pytorch-rf/artifacts/model.joblib
- deep-learning-prediction/pytorch-rf/artifacts/label_encoder.joblib
- deep-learning-prediction/pytorch-rf/artifacts/feature_columns.json
- deep-learning-prediction/pytorch-rf/artifacts/metrics.json

Results (5-fold CV on my dataset):
- Accuracy: 0.9598901098901099 (~95.99%)
- Macro-F1: 0.9587913181186317 (~95.88%)
- Per-class F1:
  - backward: 0.9572368421052632
  - forward: 0.9455782312925171
  - fowward: 0.9523809523809523   # typo exists in source labels
  - landing: 0.9655172413793104
  - left: 0.9601328903654485
  - right: 0.9570957095709571
  - takeoff: 0.9735973597359736
- Classes: ["backward","forward","fowward","landing","left","right","takeoff"]

Notes:
- Data is NOT included per project rules; only model + metrics are provided.
- Features = per-channel stats (mean/std/var/min/max/median/q25/q75/IQR/energy).
- Label “fowward” appears in the raw dataset; left as-is so reviewers can see the source state. Happy to normalize and re-train if preferred.

Addresses #306.
Closes #306

### CODE REVIEW

**Comment by 3C-SCSU** [comment]
Solved ticket: https://github.com/3C-SCSU/Avatar/issues/306
